pyqt-apps/siriushla/bl_ap_imgproc/blintlkctl.py

```diff
@@ -22,7 +22,7 @@ def beamline_opened(self):
         if self.blintlk and self.blintlk.connected:
             return self.blintlk.is_beamline_opened
         else:
-            return False
+            return 2
```

Comment:
class constant here?

Reply:
To indicate if the PV is disconnected. Because True and False are values that are returned by the 'is_beamline_opened' variable when it is connected.

Reply:
But how will the user know what `2` means? maybe in case of disconnection it would be better to raise and error, wouldn't it?

Reply:
Not really, because disconnection isn't an error, but another state of the variable. I chose the number 2 because the led widget will recognize it as the gray color automatically.